pool/connection_pool_test.go

```diff
@@ -3495,6 +3503,39 @@ func runTestMain(m *testing.M) int {
 	return m.Run()
 }
 
+func TestConnectionPool_GetInfo(t *testing.T) {
+	t.Run("equal instance info", func(t *testing.T) {
```

Comment:
Is it the only test? It confused me, because it looks similar to table driven test which should look like:

```go
var cases []struct{}{}

for _, t := range cases {
        t.Run(t.name, func(t *testing.T) {})
}
```

If I'm wrong, just resolve this conversation.

Reply:
The specification of sub tests does not require their use only in cases of table testing. In this case, it was chosen for further convenience of searching by signature

Reply:
Fine.